taxi/views.py

```diff
@@ -87,3 +89,29 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    form_class = DriverCreationForm
+    model = Driver
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
```

Comment:
check if your code is according to [checklist](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/blob/master/checklist.md)

Reply:
You about get_user_model?

Reply:
Explain please😞

Reply:
4.0 [checklist](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/blob/master/checklist.md) Use get_user_model() instead of Driver.